app/src/main/java/com/andrew/tasky/agenda/presentation/screens/event_detail/EventDetailViewModel.kt

```diff
@@ -44,10 +44,10 @@ class EventDetailViewModel @Inject constructor(
         _title.value = title
     }
 
-    private val _isAttendee = MutableStateFlow(false)
-    val isAttendee = _isAttendee.asStateFlow()
-    fun setIsAttendee(isAttendee: Boolean) {
-        _isAttendee.value = isAttendee
+    private val _isCreator = MutableStateFlow(true)
+    val isCreator = _isCreator.asStateFlow()
+    private fun setIsCreator(isCreator: Boolean) {
+        _isCreator.value = isCreator
     }
```

Comment:
Why is this function needed? It's private, so only the ViewModel can access it, but why shouldn't it change `_isCreator` directly?